lib/install/management/finder.go

```diff
@@ -269,8 +269,11 @@ func (d *Dispatcher) searchVCHsPerRP(pool *object.ResourcePool) ([]*vm.VirtualMa
 	var pools []*object.ResourcePool
 	pools = append(pools, pool)
 
-	children, err := d.listChildrenPools(pool)
+	children, err := d.session.Finder.ResourcePoolList(d.ctx, path.Join(pool.InventoryPath, "..."))
```

Comment:
`ResourcePoolList("*")` includes child pools now, this was intentional.  I believe listChildrenPools was working around this limitation.  The old Finder behavior can be forced using `"./*"` instead of `"*"`.  See the new doc on "list" vs "find" mode: https://godoc.org/github.com/vmware/govmomi/find